scripts/mintERC1155.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Script to airdrop ERC1155 tokens to a list of recipients. The list of recipients need to be passed via a JSON file.
+ * ```
+ * RECIPIENTS=<PATH> yarn hardhat run ./scripts/mintERC1155.ts --network polygon-mumbai
+ * ```
+ */
+
+import { getContractFactory, ethers, hre } from "../test/utils";
+import { readFileSync } from "fs";
+import path from "path";
+
+const TOKEN_ID = 0;
+const RECIPIENTS_CHUNK_SIZE = 100; // TODO: Still need to figure out which size is optimal
+
+async function main() {
+  const validRecipients = parseAndValidateRecipients();
+
+  const [signer] = await ethers.getSigners();
+
+  const erc1155Deployment = await hre.deployments.get("MintableERC1155");
+  const erc1155 = (await getContractFactory("MintableERC1155", { signer })).attach(erc1155Deployment.address);
+
+  for (let i = 0; i < validRecipients.length; i = i + RECIPIENTS_CHUNK_SIZE) {
+    const recipientsChunk = validRecipients.slice(i, i + RECIPIENTS_CHUNK_SIZE);
+    const airdropTx = await erc1155.airdrop(TOKEN_ID, recipientsChunk, 1);
+    console.log(
+      `Minting token with id ${TOKEN_ID} to ${recipientsChunk.length} recipients in index range ${i} - ${
+        i + RECIPIENTS_CHUNK_SIZE - 1
+      }: `,
+      airdropTx.hash
+    );
+    await airdropTx.wait();
+    console.log(`Successfully minted token to:`, recipientsChunk);
```

Comment:
Would make sense to set the token URI here too?

Reply:
I wanted to create a separate script in a separate PR that does this. Basically
- Parse and validate metadata
- Upload to IPFS
- Set token URI

Reply:
🔥